src/index.js

```diff
@@ -13,8 +13,5 @@ import FourFieldForm from './containers/FourFieldForm';
 const counterStore = createStore(reducer);
 // FLag to control which app get's rendered.
 const isExperimentalApp = true;
-if (isExperimentalApp) {
-  ReactDOM.render(<Provider store= {counterStore}> <Counter label= 'Age' /> </Provider>, document.getElementById('root'));
-} else {
-  ReactDOM.render(<FourFieldForm/>, document.getElementById('root'));
-}
+const component = isExperimentalApp ? ReactDOM.render(<Provider store= {counterStore}> <Counter label= 'Age' /> </Provider>, document.getElementById('root')) : ReactDOM.render(<FourFieldForm/>, document.getElementById('root'));
```

Comment:
Better, however you should only need to use ReactDom.render once. Can you see how you would do that?

Reply:
I am thinking of creating a function which renders the component.

Reply:
That is correct.. All you need is the single component. So you would condition this out to switch the component and then you can use ReactDom in a singular notation.
Example:
```
  const ParentComponent = isExperimentalApp ? <Counter.../> : <FormFieldForm .../>;

  ReactDOM.render(<Provider><ParentComponent ..../></Provider>, ....);
```